fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -95,7 +97,8 @@ fun AppBar(
     bottomBorder: Boolean = true,
     appTitleDelta: Float = 1.0F,
     accessoryDelta: Float = 1.0F,
-    appBarTokens: AppBarTokens? = null
+    appBarTokens: AppBarTokens? = null,
+    titleAlignment: TitleAlignment = TitleAlignment.Start
```

Comment:
I would recommend to use exisiting compose Alignment object instead of a custom TitleAlignment enum. Since now alignments are used can we allow user to End alignment or just center and start?

Reply:
updated this